### PR TITLE
Redesign search card

### PR DESCRIPTION
**Pull request by fnotorious**
## Summary

Redesigned search card

### CODE REVIEW

File: client/src/components/search/UserCard.tsx [hunk 1]
```diff
@@ -1,25 +1,57 @@
...
 export default function UserCard(props: { profile: UserProfile }) {
   return (
-    <Card direction={{ base: "column", sm: "row" }} variant="outline">
+    <Card direction={{ base: "column", sm: "row" }} p="2" variant="outline">
       <Box margin="20px" display="flex" justifyContent="center" alignItems="center">
         <Avatar mx="auto" size="xl" src={props.profile.profilePic} />
       </Box>
 
-      <Stack>
+      <Stack position="relative">
```
Comment (lesterfernandez) [inline client/src/components/search/UserCard.tsx:23]: Do we need a position of relative here?

File: client/src/components/search/UserCard.tsx [hunk 2]
```diff
@@ -1,25 +1,57 @@
...
+            <WrapItem>
+              <Tag>Co-Ed: {attributes.coed[+props.profile.coed]}</Tag>
+            </WrapItem>
+          </Wrap>
         </CardBody>
       </Stack>
 
-      <Button position="absolute" bottom="10px" right="10px" colorScheme="orange">
+      <Button
+        position="relative"
```
Comment (lesterfernandez) [inline client/src/components/search/UserCard.tsx:49]: I'm wondering the same thing about here

**Comment by lesterfernandez** [comment]
I resolved a merge conflict for you. So if you decide to change anything here you will need to pull down the most recent changes from your branch.

File: client/src/components/search/UserCard.tsx (hunk 1 above)
Comment (fnotorious) [reply client/src/components/search/UserCard.tsx:23]: My bad, force of habit

**Comment by fnotorious** [comment]
Just wanted to clarify about the minor bug fix in my commit. There was a bug where the chat buttons for some cards would be smaller than others if you minimized the screen width enough. I decided to give the chat buttons a defined minimum width.

**Review by lesterfernandez — APPROVED** [review]
Thanks for fixing the bug!

**Pull request merged**